py-polars/pyproject.toml

```diff
@@ -75,6 +75,9 @@ plot = ["hvplot >= 0.9.1", "polars[pandas]"]
 style = ["great-tables >= 0.8.0"]
 timezone = ["backports.zoneinfo; python_version < '3.9'", "tzdata; platform_system == 'Windows'"]
 
+# GPU Engine
+gpu = ["cudf-polars-cu12"]
+
 # All
 all = [
   "polars[async,cloudpickle,database,deltalake,excel,fsspec,graph,iceberg,numpy,pandas,plot,pyarrow,pydantic,style,timezone]",
```

Comment:
Question: Should `gpu` live in `all`. I think it is reasonable to do so on supported platforms once we have slimmed down the cudf-polars package.

Reply:
Probably not.

Looking at the size of libcudf on Anaconda it's 360 MB which is pretty large in of itself.

Reply:
Agreed, let's leave it out.